nss-day-cohort-60/rare-python-server-tyrannical-tators
# Create Subscription

#Closes #58

# Description 

Given the client clicks the subscribe button in the browser to send a request to create a new user subscription
When a valid subscription object is sent to the API
Then the API should create a new subscription
And the new subscription should appear in the subscriptions table

Created subscription in model.py, create function in repository, and updated do_POST method in request_handler.

# Steps to Test

1. Fetch and checkout to branch [[create-subscription-MP]]
2. Open VSCode, open Sqlite, and start you debugger
3. Open Postman
4. Send a post request to http://localhost:8088/subscriptions such as {
    "follower_id": 3,
    "author_id": 2,
    "created_on": "2023-01-28 14:29:58.811467"
}
5. Header should be 201, data sent should appear in response body
6. Verify that the data was saved to your local database by checking the subscriptions table in your database
